src/js/components/Popup/PopupTooltip.tsx

```diff
@@ -19,7 +25,9 @@ const PopupTooltip = ({
   mods,
   textHighlightColor,
   children,
+  variant = PopupTooltipVariant.DARK,
 }: Props) => {
+  console.log(variant);
```

Comment:
console.log

Reply:
I keep forgetting to remove these. Thanks!